webapp/channels/package.json

```diff
@@ -198,7 +198,7 @@
     "clean": "rm -rf dist node_modules .eslintcache .stylelintcache tsconfig.tsbuildinfo",
     "stats": "cross-env NODE_ENV=production webpack --profile --json > webpack_stats.json",
     "mmjstool": "mmjstool",
-    "i18n-extract": "npm run mmjstool -- i18n extract-webapp --webapp-dir ./src",
+    "i18n-extract": "npm run mmjstool -- i18n extract-webapp --webapp-dir ./src && ./gen_lang_imports",
```

Comment:
Why call this in i18n-extract? Do we expect devs or CI have to call this automatically, or could we just have it as a separate script that we call manually when a new language is added?

Reply:
Mostly wanted to make it as simple as possible to get the generated file updated in case a new language file is added. We can have a separate script though, as I don't think it would happen that often.

Reply:
Yeah, I'd like it to be a separate script. This script is called by CI to verify that the translation files are in sync with the web app, and it's a bit weird to also have it do that. I think it'd also cause any PR which adds a language to fail CI